### PR TITLE
Update Traditional Chinese translation

### PR DESCRIPTION
**Pull request by ianiiaannn**
Fixed the formatting issues metioned in #54 by replace U+FF1A after replacing regular colon.
Screenshot 2023-06-24 012220
Screenshot 2023-06-24 012244
Screenshot 2023-06-24 012305

### CODE REVIEW

**Comment by flandretw** [comment]
@ianiiaannn
I agree to this translation formatting correction. And can you help me change my name to flandretw? That's my new name, thank you very much.

**Comment by Intelli** [comment]
The problem with replacing the colon with a separate space (": "), with the "：" character, is that the backend translation processing on the web server will then replace it with an ":" character... but then we'll be missing a space.

So ideally, we should just keep a standard colon. Rest of the changes should be fine though.

File: lang/zh-tw.yml [hunk 1]
```diff
@@ -1,185 +1,185 @@
...
 
-ACTION_NOT_SUPPORTED: "此指令不支援該操作。"
-AMOUNT_BLOCK: "{0} 個{方塊|方塊}"
+ACTION_NOT_SUPPORTED: "此指令不支援該行為。"
+AMOUNT_BLOCK: "{0} 格{方塊|方塊}"
 AMOUNT_CHUNK: "{0} 個{區塊|區塊}"
 AMOUNT_ENTITY: "{0} 個{實體|實體}"
 AMOUNT_ITEM: "{0} 個{物品|物品}"
 API_TEST: "API 測試成功。"
-CACHE_ERROR: "警告: 驗證快取 {0} 時出現錯誤。"
```
Comment (Intelli) [inline lang/zh-tw.yml:10]: Colons shouldn't be modified.

File: src/main/java/net/coreprotect/language/Phrase.java [hunk 2]
```diff
@@ -265,7 +266,8 @@ public static String build(Phrase phrase, String... params) {
         }
 
         if (color.length() > 0) {
-            output = output.replaceFirst(split, split + color);
+            output = output.replaceFirst(SPLIT, SPLIT + color);
+            output = output.replaceFirst(FULL_WIDTH_SPLIT, FULL_WIDTH_SPLIT + color);
```
Comment (Intelli) [inline src/main/java/net/coreprotect/language/Phrase.java:270]: You can keep these changes, for people who wish to run their own modified language files utilizing the FULL_WIDTH_SPLIT (colon) character.

**Review by Intelli — CHANGES_REQUESTED** [review]

**Pull request merged**